test/facade_converter_test.ts

```diff
@@ -232,6 +232,15 @@ var y = x.length;`);
   var x = new RegExp(r'a');
   x.hasMatch("a");
 }`);
+    expectWithTypes('function f() { var x = /a(.)/g; x.exec("ab")[1]; }').to.equal(`f() {
+  var x = new RegExp(r'a(.)');
```

Comment:
Actually, now fixed the existing e2e test that initially came up red in travis :-)